documentation/modules/oauth/con-oauth-authorization-keycloak-authorization-services.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following module:
+//
+// proc-oauth-authorization-broker-config.adoc
+
+[id='con-oauth-authorization-keycloak-authorization-services_{context}']
+= Manage policies and permissions in Keycloak Authorization Services
+
+== Kafka Authorization Model
+
+The link:https://kafka.apache.org/documentation/#security_authz_primitives[Kafka authorization model] defines resource types, and permissions available for each type.
+Whenever some action is performed by the Kafka client, one or more permissions are checked by the broker (depending on the action performed) using the configured authorizer.
+
+There are five resource types: `Topic`, `Group`, `Cluster`, `TransactionalId`, `DelegationToken`.
+
+And these are the different permissions for each resource type:
```

Comment:
```suggestion
Each resource type has different permissions:
```

Reply:
Are these the permissions we need to capture in Keycloak Authorization Services? If so, maybe we should say that. _Permissions for each resource type, based on operations, must be captured in the Keycloak Authorization Services Model_

Reply:
How much are we likely to have to update these lists? Would it be accurate to say something like _make sure you capture permissions for all the supported operations of each resource_?